packages/mdc-fab/README.md

```diff
@@ -59,51 +59,15 @@ We recommend you load [Material Icons](https://material.io/icons/) from Google F
 
 > _NOTE:_ The floating action button icon can be used with a `span`, `i`, `img`, or `svg` element.
 
-### CSS Classes
-
-CSS Class | Description
---- | ---
-`mdc-fab` | Mandatory, for the button element
-`mdc-fab__icon` | Mandatory, for the icon element
-`mdc-fab--mini` | Optional, modifies the FAB to a smaller size
-`mdc-fab--exited` | Optional, animates the FAB out of view.<br>When this class is removed, the FAB will return to view.
-
-> **A note about `:disabled`**, No disabled styles are defined for FABs. The FAB promotes action, and should not be displayed in a disabled state. If you want to present a FAB that does *not* perform an action, you should also present an explanation to the user.
-
-### Absolutely Positioned
+### Styles
 
-Developers must position MDC FAB as needed within their application's design.
-
-```html
-<!--
-  This will position the FAB in the bottom-right corner.
-  Modify to fit your design's requirements.
--->
-<style>
-.app-fab--absolute {
-  position: fixed;
-  bottom: 1rem;
-  right: 1rem;
-}
-
-@media(min-width: 1024px) {
-   .app-fab--absolute {
-    bottom: 1.5rem;
-    right: 1.5rem;
-  }
-}
-</style>
-<button class="mdc-fab material-icons app-fab--absolute" aria-label="Favorite">
-  <span class="mdc-fab__icon">
-    favorite
-  </span>
-</button>
+```scss
+@import "@material/fab/mdc-fab";
 ```
 
-### Adding MDC Ripple
+### JavaScript Instantiation
 
-To add the ripple effect to an MDC FAB, attach a [ripple](../mdc-ripple) instance to the
-`mdc-fab` element.
+The FAB will work without JavaScript, but you can enhance it to have a ripple effect by instantiating `MDCRipple` on the root element. See [MDC Ripple](../mdc-ripple) for details.
```

Comment:
Make this section match the Button readme

* JS example should show import and application, and prefer `new` over `attachTo`
* Remove the declarative paragraphs/example